app/presenters/tasks/generic.rb

```diff
@@ -24,6 +24,14 @@ def completed?(rec = record, form = associated_form)
 
     private
 
+    def previous_tasks
+      if self.class.const_defined?(:PREVIOUS_TASKS)
+        self.class::PREVIOUS_TASKS
+      else
+        [self.class::PREVIOUS_TASK]
+      end
+    end
```

Comment:
not necessarily in this PR, but had been wanting to rename/pluralise all PREVIOUS_TASK to PREVIOUS_TASKS and enforce as always being an array, even if only one element. This could make this logic terser and more consistent.